src/main/xerus/monstercat/api/response/Track.kt

```diff
@@ -53,15 +50,15 @@ open class Track: MusicItem() {
 		if(titleClean.isNotEmpty())
 			return this
 		
-		if(::release.isInitialized) {
+		/*if(::release.isInitialized) { TODO : Remove this as it's useless, unless proven
```

Comment:
are you sure?

Reply:
Linked to nothing.

Reply:
Linked? You mean it was never called?

Reply:
Yeah, only in TabDownloader, where it is set but never read.

Reply:
You can add it back, but you'll have to deal with a crash I couldn't  find the cause

Reply:
Have you sent me the stacktrace? I never got a crash.